frontend/src/pages/studentLesson/QuestionAttempts.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import { useParams } from 'react-router-dom';
+import { makeStyles } from '@material-ui/core/styles';
+import { DataGrid } from '@material-ui/data-grid';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Grid from '@material-ui/core/Grid';
+import Modal from '@material-ui/core/Modal';
+
+import Title from '../_shared/Title';
+import service from '../../service';
+
+const convertDateToReadableString = (date) => {
+  return `${date.substring(0, 10)} ${date.substring(11, 19)}`;
+};
+
+const columns = [
+  {
+    field: 'updatedAt',
+    headerName: 'Datum',
+    flex: 0.2,
+    renderCell: (params) => convertDateToReadableString(params.row.updatedAt),
+  },
+  {
+    field: 'questionGroup',
+    headerName: 'Vraag groep',
+    flex: 0.4,
+    valueGetter: (params) => params.row.Question.questions.name,
+  },
+  {
+    field: 'questionGroupIndex',
+    headerName: 'Vraag groep index',
+    flex: 0.2,
+    valueGetter: (params) => params.row.Question.questions.index + 1,
+  },
+  {
+    field: 'question',
+    headerName: 'Vraag nummer',
+    flex: 0.2,
+    valueGetter: (params) => params.row.Question.index + 1,
+  },
+];
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    position: 'absolute',
+    width: 400,
+    backgroundColor: theme.palette.background.paper,
+    border: '2px solid #000',
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing(2, 4, 3),
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+  },
+}));
+
+const Activity = (props) => {
+  const classes = useStyles();
+  const params = useParams();
+  const [questionAttempts, setQuestionAttempts] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  const [modalOpen, setModalOpen] = useState(false);
+  const [modalContent, setModalContent] = useState({});
+
+  const loadQuestionAttempts = useCallback(async () => {
```

Comment:
would be nice to put this stuff in the questionStore

Reply:
Added new task